From 62abb3465b7f4ca3d9aab59b4c5b17a2c2026cbe Mon Sep 17 00:00:00 2001
From: Dien Pham <dien.pham.ry@renesas.com>
Date: Thu, 16 Nov 2017 11:58:07 +0700
Subject: [PATCH 574/909] arm64: dts: salvator-common: Add cpu-supply property
 in a57_0 node

commit c926af61e9a0975c6228bffc7d76d475e46a9a12 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

In order to validate CPU Freq, this patch adds cpu-supply property to
a57_0 node.

Signed-off-by: Dien Pham <dien.pham.ry@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index 54a920be16fd..926ead755385 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -291,6 +291,10 @@
 	};
 };
 
+&a57_0 {
+	cpu-supply = <&dvfs>;
+};
+
 &audio_clk_a {
 	clock-frequency = <22579200>;
 };
-- 
2.17.1

